### DISCUSSION TITLE
post/2023/2023-03-configuring-forward-and-inverse-search-in-latex-workshop-and-okular/

### DISCUSSION THREAD

**Discussion by giscus** [zyrikby/website_related#26] [Announcements]
# post/2023/2023-03-configuring-forward-and-inverse-search-in-latex-workshop-and-okular/

Although many researchers now prefer to write their papers using various SaaS solutions, such as Overleaf, I still promote an old-style way when everything is installed on your computer. Of course, this approach has some pros and cons. However, in this article, I am not going to discuss them and will concentrate instead on the topic of how to configure forward and inverse search. In particular, I will show how to do this for my setup with LaTeX Workshop, a VS Code extension facilitating text writing in TeX, and Okular, a PDF viewer available for Linux and Windows platforms.

https://zhauniarovich.com/post/2023/2023-03-configuring-forward-and-inverse-search-in-latex-workshop-and-okular/

**Comment by JackYansongLi** [comment]
Wow! you saved my life!
